e2e-tests/desktop/playwright.config.js

```diff
@@ -6,6 +6,18 @@ import { defineConfig } from '@playwright/test';
 
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 export default defineConfig({
+  globalSetup: '../global-setup',
   outputDir: './artifacts',
   workers: 1, // Tests need to be run in serial, otherwise there may be conflicts when using the CLI
+  use: {
+    baseURL: process.env.BOUNDARY_ADDR,
+    storageState: './admin/artifacts/authenticated-state.json',
+    extraHTTPHeaders: {
+      // This token is set in global-setup.js
+      Authorization: `Bearer ${process.env.E2E_TOKEN}`,
+    },
+  },
+  expect: {
+    timeout: 10000,
```

Comment:
is there any difference in using timeout within expect vs using `timeout` option in admin config? 

`  timeout: 90000, // Each test is given 90s to complete`

Reply:
Yes, this is for specifically the timeout of expect statements, while the timeout at the root is for an overall test timeout.